tools/pypsa/features.yaml

```diff
@@ -45,82 +66,127 @@ features:
-      source: []
-      value: n
+      source:
+        - https://docs.pypsa.org/latest/user-guide/components/generators/
+        - https://docs.pypsa.org/latest/user-guide/components/links/
+        - https://docs.pypsa.org/latest/user-guide/components/stores/
+        - https://docs.pypsa.org/latest/user-guide/components/storage-units/
+        - https://docs.pypsa.org/latest/user-guide/components/lines/
+        - https://docs.pypsa.org/latest/user-guide/components/transformers/
+      value: y
     static_dimensional:
-      source: []
-      value: n
+      source:
+        - https://docs.pypsa.org/latest/user-guide/components/generators/
+        - https://docs.pypsa.org/latest/user-guide/components/links/
+        - https://docs.pypsa.org/latest/user-guide/components/stores/
+        - https://docs.pypsa.org/latest/user-guide/components/storage-units/
+        - https://docs.pypsa.org/latest/user-guide/components/lines/
+        - https://docs.pypsa.org/latest/user-guide/components/transformers/
+      value: y
   asset__operating_constraints:
     inter_spatial:
-      source: []
-      value: n
+      source:
+        - https://docs.pypsa.org/latest/user-guide/design/#buses
+        - https://docs.pypsa.org/latest/user-guide/components/lines/
+        - https://docs.pypsa.org/latest/user-guide/components/links/
+      value: y
     inter_temporal:
       value: y
       source:
-        - https://pypsa.readthedocs.io/en/latest/user-guide/optimal-power-flow.html#ramping-constraints
-        - https://pypsa.readthedocs.io/en/latest/examples/unit-commitment.html#Ramp-rate-limits
+        - https://docs.pypsa.org/latest/user-guide/optimization/unit-commitment/#ramping
+        - https://docs.pypsa.org/latest/examples/unit-commitment/#ramp-rate-limits
     operating:
       value: y
       source:
-        - https://pypsa.readthedocs.io/en/latest/examples/unit-commitment.html#Minimum-part-load-demonstration
-        - https://pypsa.readthedocs.io/en/latest/examples/unit-commitment.html#Minimum-up-time-demonstration
-        - https://pypsa.readthedocs.io/en/latest/examples/unit-commitment.html#Minimum-down-time-demonstration
+        - https://docs.pypsa.org/latest/examples/unit-commitment/#minimum-part-load
+        - https://docs.pypsa.org/latest/examples/unit-commitment/#minimum-up-time
+        - https://docs.pypsa.org/latest/examples/unit-commitment/#minimum-down-time
     total:
-      source: []
-      value: n
+      source:
+        - https://docs.pypsa.org/latest/examples/simple-electricity-market-examples/
+        - https://docs.pypsa.org/latest/user-guide/design/#buses
+        - https://docs.pypsa.org/latest/user-guide/components/lines/
+        - https://docs.pypsa.org/latest/user-guide/components/links/
+        - https://docs.pypsa.org/latest/user-guide/optimization/custom-constraints/#custom-constraints
+      value: y
   asset__operating_representation:
     ed:
       source: []
       value: n
```

Comment:
If you can do UCED, you can do pure economic dispatch (i.e. when you group assets there is no concept of committing units any more). UCED requires MILP, ED doesn't. I have been thinking of removing this tbh as ED is a baseline and UCED is kinda the "feature" (as with multi/single carrier models later on)

Reply:
The only harm is in the size of the feature list, but yes I see what you mean